src/components/about/ContactCard.js

```diff
@@ -101,56 +96,45 @@ const ModalCard = ({ imageUrl, imageAlt, modalTitle, title, blurb }) => {
         as="a"
         href="#"
         ref={focusRef}
-        maxW={[null, '100%', '342px']}
+        margin="5% auto"
```

Comment:
Couple things here. as a rule I prefer to use em or rem values for margin to keep things flexible. Percentages tend to be a bit unreliable. We have a `theme.spacing` variable we have access to that helps to use the same values across the site, and prevent us from having tons of different specific pixel or percentage values. try `theme.spacing.base` here and see how it looks.

Might be able to do some sneaky JS with the margin here so you don't have to redefine marginBottom below:

`margin={ [theme.spacing.base, 'auto', margin || theme.spacing.base, 'auto'] }`

I _think_ that should work 😉

Reply:
what is margin here, an actual value? When implementing the suggestion, we get margin is undefined, so maybe I'm missing something.

Reply:
@juanri0s interesting. We may need to set the `margin || theme.spacing.base` value to it's own variable.

Presumably this would be a value that Chakra can ingest, it takes string values like `"1rem"` or numeric values like `0`.

I don't want this to hold you up too much we can also remove the need for the margin prop and just adjust this manually if we need!